servicemigration/context.py

```diff
@@ -41,7 +41,10 @@ def __init__(self, filename=None):
         else:
             infile = os.environ["MIGRATE_INPUTFILE"] if "MIGRATE_INPUTFILE" in os.environ else None
 
-        self._client = ControlPlaneClient(**getConnectionSettings())
+	if ZenUtils:
```

Comment:
Is this a functional change unrelated to cleanup, or something you needed to run the `test` target?